src/ansys/fluent/core/solver/flobject.py

```diff
@@ -161,11 +161,11 @@ def get_attr(self, attr, attr_type_or_types=None) -> Any:
                 attr_type_or_types = (attr_type_or_types,)
             if isinstance(val, attr_type_or_types):
                 return val
-            if val is not None and any(
+            if val != [] and any(
                 issubclass(x, bool) for x in attr_type_or_types
             ):  # cast to bool for boolean attributes
                 return bool(val)
-            return None
+            return []
```

Comment:
We want to return '[ ]` instead of 'None` in following case right ? 

![image](https://github.com/ansys/pyfluent/assets/106588300/e862277d-6731-48f0-85f7-dca77bbb9844)

Reply:
What about other attributes? For example, display-text?

Reply:
We have restored original changes. We will discuss this soon.